File: Codes/Uri1043.java
// "Perimetro = "
// "Area = "
// 6,0 4,0 2,0

import java.util.Scanner;

public class Uri1043 {

    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        double a = s.nextDouble();
        double b = s.nextDouble();
        double c = s.nextDouble(); 
        s.close();   
        if ((a < b + c) && (b < a + c) && (c < b + a)) {
            double perimetro = a + b + c;
            System.out.printf("Perimetro = %.1f\n", perimetro);
      } else {
            double area = ((a + b) * c) / 2;
            System.out.printf("Area = %.1f\n", area);
        }
    }
}

File: Codes/Uri1043.js
const Uri1043 = function Uri1043(a, b, c) {
    if ((a < b + c) && (b < a + c) && (c < b + a)) {
        let perimetro = a + b + c
        console.log("Perimetro =", perimetro)
    } else {
        let area = ((a + b) * c) / 2
        console.log("Area = " + area)
    }
}

Uri1043(6.0, 4.0, 2.0)